tests/iac/test_filter.py

```diff
@@ -66,7 +69,10 @@ def test_get_iac_files_from_paths_ignore_git(tmp_path):
     subprocess.run(["git", "init"], cwd=tmp_path_str)
     subprocess.run(["git", "add", "."], cwd=tmp_path_str)
 
-    files = get_iac_files_from_paths(tmp_path, set(), True)
-    assert len(files.files) == 9
-    assert str("file2.json") not in files.files
-    assert str("file3.yaml") in files.files
+    files = get_iac_files_from_paths(tmp_path, set(), True, ignore_git)
+    if ignore_git:
+        assert len(files.files) == 10
```

Comment:
nit: maybe add `assert "file2.json" in files.files` here to make the comparison with the `else` case even clearer, if you feel it's redundant, it's not very important of course.